mappings/net/minecraft/util/registry/Registry.mapping

```diff
@@ -1,33 +1,33 @@
 CLASS fk net/minecraft/util/registry/Registry
 	FIELD A CHUNK_STATUS Lex;
-	FIELD B STRUCTURE_FEATURES Lfk;
-	FIELD C STRUCTURE_PIECES Lfk;
+	FIELD B STRUCTURE_FEATURE Lfk;
+	FIELD C STRUCTURE_PIECE Lfk;
 	FIELD D RULE_TEST Lfk;
 	FIELD E STRUCTURE_PROCESSOR Lfk;
 	FIELD F STRUCTURE_POOL_ELEMENT Lfk;
-	FIELD G STAT_TYPES Lfk;
+	FIELD G STAT_TYPE Lfk;
 	FIELD e LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD f REGISTRIES Lfo;
-	FIELD g SOUNDS Lfk;
-	FIELD h FLUIDS Lex;
-	FIELD i POTION_EFFECT_TYPES Lfk;
-	FIELD j BLOCKS Lex;
-	FIELD k ENCHANTMENTS Lfk;
-	FIELD l ENTITY_TYPES Lfk;
-	FIELD m ITEMS Lex;
-	FIELD n POTIONS Lfk;
-	FIELD o CARVERS Lfk;
-	FIELD p SURFACE_BUILDERS Lfk;
-	FIELD q FEATURES Lfk;
-	FIELD r DECORATORS Lfk;
-	FIELD s BIOMES Lfk;
-	FIELD t PARTICLE_TYPES Lfk;
-	FIELD u BIOME_SOURCE_TYPES Lfk;
-	FIELD v BLOCK_ENTITIES Lfk;
-	FIELD w CHUNK_GENERATOR_TYPES Lfk;
-	FIELD x DIMENSIONS Lfk;
-	FIELD y MOTIVES Lex;
-	FIELD z CUSTOM_STATS Lfk;
+	FIELD g SOUND Lfk;
```

Comment:
Unsure about this. The registry name is `sound_event`, but we use `Sound` for the class name - this should change, since it isn't an actual sound, it's the event that plays *a* sound.

Reply:
It should change to SoundEvent then

Reply:
I agree change the class + this field

Reply:
Yes.